yt_dlp/downloader/http.py

```diff
@@ -202,14 +197,16 @@ def establish_connection():
                     # Unexpected HTTP error
                     raise
                 raise RetryDownload(err)
-            except socket.timeout as err:
-                raise RetryDownload(err)
-            except socket.error as err:
-                if err.errno in (errno.ECONNRESET, errno.ETIMEDOUT):
-                    # Connection reset is no problem, just retry
+            except compat_urllib_error.URLError as err:
+                # TODO: What errors should we exclude?
+                if not any(isinstance(err.reason, et) for et in (ssl.CertificateError, )):
```

Comment:
Huh, shouldn't there be something after that comma?

Reply:
I was going to add more but couldn't think of any. Just left it like that for now 😅

Reply:
isinstance takes multiple items
```py
                if not isinstance(err.reason, (ssl.CertificateError, ...)):
```